### PR TITLE
feat: Configuring certificate issuers for apps

### PR DESCRIPTION
**Pull request by gvicentin**

### CODE REVIEW

File: api/server.go [hunk 1]
```diff
@@ -305,6 +305,8 @@ func RunServer(dry bool) http.Handler {
 	m.Add("1.2", http.MethodGet, "/apps/{app}/certificate", AuthorizationRequiredHandler(listCertificates))
 	m.Add("1.2", http.MethodPut, "/apps/{app}/certificate", AuthorizationRequiredHandler(setCertificate))
 	m.Add("1.2", http.MethodDelete, "/apps/{app}/certificate", AuthorizationRequiredHandler(unsetCertificate))
+	m.Add("1.0", http.MethodPut, "/apps/{app}/certissuer", AuthorizationRequiredHandler(setCertIssuer))
```
Comment (wpjunior) [inline api/server.go:308]: @gvicentin we may change "1.0" to future version of tsuru api.

File: app/actions.go [hunk 2]
```diff
@@ -694,3 +696,124 @@ var rebuildRoutes = action.Action{
...
+		if err != nil {
+			log.Errorf("BACKWARD remove certissuer db. unable to connect: %s", err)
+			return
+		}
+		defer conn.Close()
+
+		sanitizedCName := strings.ReplaceAll(cname, ".", "_dot_")
+		certIssuerCName := fmt.Sprintf("certissuers.%s", sanitizedCName)
+
+		err = conn.Apps().Update(
```
Comment (wpjunior) [inline app/actions.go:760]: @gvicentin we could rebase to update using new mongo driver.

Comment (gvicentin) [reply app/actions.go]: Thanks! Fixed ✅

File: api/server.go (hunk 1 above)
Comment (gvicentin) [reply api/server.go]: Thanks! Fixed ✅

**Review by wpjunior — APPROVED** [review]

**Pull request merged**